include/groonga/table.h

```diff
@@ -500,6 +500,17 @@ GRN_API grn_rc
 grn_index_cursor_reset_start_position(grn_ctx *ctx, grn_obj *index_cursor);
 GRN_API uint32_t
 grn_index_cursor_get_start_position(grn_ctx *ctx, grn_obj *index_cursor);
+/**
+ * \brief Retrieve the posting information and move the cursor to the the next.
+ *
+ * \param ctx The context object.
+ * \param index_cursor Target cursor.
+ * \param term_id The buffer to store the ID of the current target record of
+ *                cursor. If NULL is specified, it will not be stored.
+ *
+ * \return \ref grn_posting if the next posting is found, NULL if not.
+ *         Return value \ref grn_posting need not be freed.
```

Comment:
主語をYouにしちゃった方がシンプルかも。

```suggestion
 *         You don't need to free the returned \ref grn_posting.
```

Reply:
タシカニ。